test/e2e/services.go

```diff
@@ -174,6 +186,27 @@ func newLokiService(e e2e.Environment) *e2e.InstrumentedRunnable {
 	)
 }
 
+func newRulesBackendService(e e2e.Environment) *e2e.InstrumentedRunnable {
+	ports := map[string]int{"http": 8080, "internal": 8081}
+
+	args := e2e.BuildArgs(map[string]string{
+		"-log.level":            logLevelDebug,
```

Comment:
Is the object store using `-` for long options instead of `--`?

Reply:
added `--` to be consistent :)